### PR TITLE
Update alarm doc introduce keys with data types of the configuration file.

### PR DESCRIPTION
**Pull request by pg-yang**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: docs/en/setup/backend/backend-alarm.md [hunk 1]
```diff
@@ -348,3 +348,23 @@ which will override the settings in `alarm-settings.yml`.
 In order to determine whether an alerting rule is triggered or not, SkyWalking needs to cache the metrics of a time window for
 each alerting rule. If any attribute (`metrics-name`, `op`, `threshold`, `period`, `count`, etc.) of a rule is changed,
 the sliding window will be destroyed and re-created, causing the Alarm of this specific rule to restart again.
+
+### Configuration property key of Alerting element
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-alarm.md:352]: What is this doc about? What property keys? Could you point to the source codes about this?

Comment (pg-yang) [reply docs/en/setup/backend/backend-alarm.md:352]: [RulesReader](https://github.com/apache/skywalking/blob/master/oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java#L85)

There is no doc list these key

Comment (pg-yang) [reply docs/en/setup/backend/backend-alarm.md:352]: I haven searched  these key in all MD doc.

Comment (wu-sheng) [reply docs/en/setup/backend/backend-alarm.md:352]:
```suggestion
### Keys with data types of alerting rule configuration file
```

**Review by wu-sheng — APPROVED** [review]

**Comment by pg-yang** [comment]
Need improve English . Haha

**Pull request merged**